### DISCUSSION TITLE
Summary plot not displaying all the anatomical sites associated with HMP data

### DISCUSSION THREAD

**Discussion by nsuvarnaiari** [nih-cfde/published-documentation#190] [Q&A] [answered]
While reviewing HMP's latest submission, I have noticed that the summary plot under "Contents Summary Charts" of "In-Review Submission" menu option, has only 13 out of 27 anatomical sites and all other sites are binned under “other”. However, I  see the correct number of anatomical sites in the summary and also in the “refine search” panel under anatomy. Is that a known issue with the display of data or something wrong with the submission?

![Screen Shot 2021-03-15 at 6 40 06 PM](https://user-images.githubusercontent.com/13201743/111230571-35555f80-85be-11eb-812d-c82b0094edd8.png)

**Comment by ACharbonneau** [comment, marked as answer]
In many cases, the number of easily discernable colors we can use to plot is fewer than the possible number of options. Similarly, we have designed the website to work on both desktop computers and tablets, which limits the maximum plot width, and with it the number of columns that can be shown before the axis label display becomes too difficult to read. 

The current behavior of this plot (and similar plots throughout the portal) is to display up to 13 options as either colors or bars. If an option has 14+ options, then the plot will show the 12 options with the highest values, and then to bin all other options into 'other' with the 13th color. This way, the plots always display properly, even for extremely variable datasets. 

We are always looking for new ways to display data effectively, so if you know of plot types that might better display this type of data, please make a feature request.
